### PR TITLE
jellyseerr: 1.4.1 -> 1.7.0

### PR DESCRIPTION
**Pull request by camillemndn**
## Description of changes

Changelog: https://github.com/Fallenbagel/jellyseerr/releases/tag/v1.7.0
Based on https://github.com/NixOS/nixpkgs/pull/250412 thanks to @Freakmiko 
I don't understand if this addresses https://github.com/NixOS/nixpkgs/issues/258048?

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by JulienMalka — APPROVED** [review]

File: pkgs/servers/jellyseerr/package.json [hunk 1]
```diff
@@ -89,42 +89,41 @@
     "secure-random-password": "0.2.3",
     "semver": "7.3.8",
     "sqlite3": "5.1.4",
-    "swagger-ui-express": "4.6.0",
-    "swr": "2.0.0",
-    "typeorm": "0.3.11",
+    "swagger-ui-express": "4.6.2",
+    "swr": "2.0.4",
+    "typeorm": "0.3.12",
```
Comment (Stunkymonkey) [inline pkgs/servers/jellyseerr/package.json:94]: reading the release notes it states:
`Before you yarn build please downgrade your typeorm version by yarn add typeorm@0.3.11`

Comment (Freakmiko) [reply pkgs/servers/jellyseerr/package.json:94]: Right above that it says "Warning (For windows users)" so this shouldn't be of concern, right?

Comment (Stunkymonkey) [reply pkgs/servers/jellyseerr/package.json:94]: sorry, you are right.

**Comment by Stunkymonkey** [comment]
Result of `nixpkgs-review pr 259076` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>jellyseerr</li>
  </ul>
</details>

**Review by Stunkymonkey — APPROVED** [review]

**Pull request merged**